bot.py

```diff
@@ -431,33 +433,102 @@ def main_color(self):
             )
         logger.info(LINE)
 
-    async def process_modmail(self, message):
-        """Processes messages sent to the bot."""
-
+    async def retrieve_emoji(self):
         ctx = SimpleNamespace(bot=self, guild=self.modmail_guild)
         converter = commands.EmojiConverter()
 
-        blocked_emoji = self.config.get('blocked_emoji', '🚫')
         sent_emoji = self.config.get('sent_emoji', '✅')
+        blocked_emoji = self.config.get('blocked_emoji', '🚫')
+
+        if sent_emoji not in UNICODE_EMOJI:
+            try:
+                sent_emoji = await converter.convert(
+                    ctx, sent_emoji.strip(':')
+                )
+            except commands.BadArgument:
+                logger.warning(info(f'Sent Emoji ({sent_emoji}) '
+                                    f'is not a valid emoji.'))
+                del self.config.cache['sent_emoji']
+                await self.config.update()
 
         if blocked_emoji not in UNICODE_EMOJI:
             try:
                 blocked_emoji = await converter.convert(
                     ctx, blocked_emoji.strip(':')
                 )
             except commands.BadArgument:
-                pass
+                logger.warning(info(f'Blocked emoji ({blocked_emoji}) '
+                                    'is not a valid emoji.'))
+                del self.config.cache['blocked_emoji']
+                await self.config.update()
+        return sent_emoji, blocked_emoji
 
-        if sent_emoji not in UNICODE_EMOJI:
+    async def process_modmail(self, message):
+        """Processes messages sent to the bot."""
+        sent_emoji, blocked_emoji = await self.retrieve_emoji()
+
+        account_age = self.config.get('account_age')
+        if account_age is None:
+            account_age = isodate.duration.Duration()
```

Comment:
`account_age = config.get() or isodate.duration.Duration()`

Reply:
`config.get('account_age', isodate.duration.Duration())`

Reply:
I did that for the else clause directly follows that